src/invidious/config.cr

```diff
@@ -222,6 +238,24 @@ class Config
         end
     {% end %}
 
+    if !config.invidious_companion.empty?
```

Comment:
Codestlye nitpick

What do you think about replacing `!empty?` with `present?` instead or `unless {...}.empty?`

I've seen a couple discussions in the Crystal community regarding `if !empty?` being harder to process cognitively  due to an essentially double negation.

Related discussions:
https://forum.crystal-lang.org/t/collections-any-vs-empty/5303
https://github.com/crystal-lang/crystal/issues/13847
https://github.com/crystal-lang/shards/pull/577#discussion_r1088439033

Reply:
That's perfect for me. That was already odd for me to do `!empty?`, happy there is an alternative.